core/templates/.spacemacs.template

```diff
@@ -62,6 +62,8 @@ values."
    ;; environment, otherwise it is strongly recommended to let it set to t.
    ;; (default t)
    dotspacemacs-elpa-https t
+   ;; If non nil then checking for updates is enabled (default t)
```

Comment:
Can you add a `.` at the end of the sentence, before `(default t)` for consistency with other ones? :smile: